com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/editor/JsonScanner.java

```diff
@@ -68,7 +67,7 @@ protected void init() {
                 PreferenceConstants.ITALIC_KEY, PreferenceConstants.UNDERLINE_KEY);
         IToken keyToken = new YAMLToken(keyAttr, YAMLToken.KEY);
 
-        TextAttribute pathKeyAttr = tokenAttribute(KaizenPreferenceConstants.COLOR_PATH, PreferenceConstants.BOLD_KEY,
```

Comment:
So, now `KaizenPreferenceConstants.COLOR_PATH` is set in PreferencesInitializer, but it's never used, is that right? It's confusing without knowing the history of changes. I suggest removing it or, if we want to keep it for future, add a comment that it was created to be used for `pathKeyAttr ` in `JsonScanner` and a link to the discussion in PR.